alacritty_terminal/src/term/mod.rs

```diff
@@ -47,7 +47,7 @@ pub mod cell;
 pub mod color;
 
 // De facto limit - see https://stackoverflow.com/a/417184/
```

Comment:
Small nit: since you're changing the max length away from  the "de facto" proposed in that overflow answer, would it be a good idea to remove this comment?

Reply:
Oh, yep!